routes/response-route.js

```diff
@@ -0,0 +1,12 @@
+var express = require "express";
+var responseRouter  = express.Router();
+var Question = require('../models/question.js');
+
+
+responseRouter.get('/',(req,res)=>{
+    response.findAll().then((responses)=>{
+        res.send(responses)
+    })
+})
+
```

Comment:
right idea, but need to use the model to findAll. In this case we would need to import in the Response model (instead of Question) and use it like this:

```js
var Response = require('../models/response.js');

responseRouter.get('/',(req,res)=>{
     Response.findAll().then((responses)=>{
         res.send(responses)
     })
 })
```